# Description: Do a better job of handling latex special entities.
#   Mike's original problem was handling special << >> quote characters, i.e.
#   from a French keyboard.  He solved this problem by adding the fontenc
#   package.  In testing, I found that this change caused a HUGE slowdown for
#   PostScript and PDF output, i.e. on the order of several minutes for dvips
#   to render the result.  After digging around on Google, I added the lmodern
#   package and that seems to have solved the problems I observed.
# Bug-Debian: http://bugs.debian.org/614570
# Author: Kenneth J. Pronovici <pronovic@debian.org>
# Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/epydoc/docwriter/latex.py
+++ b/epydoc/docwriter/latex.py
@@ -29,6 +29,8 @@
         "\\documentclass{article}",
         "\\usepackage{alltt, parskip, fancyhdr, boxedminipage}",
         "\\usepackage{makeidx, multirow, longtable, tocbibind, amssymb}",
+        "\\usepackage[T1]{fontenc}",
+        "\\usepackage{lmodern}",
         "\\usepackage{fullpage}",
         "\\usepackage[usenames]{color}",
         # Fix the heading position -- without this, the headings generated
